packages/compass-saved-aggregations-queries/src/hooks/use-grid-filters.tsx

```diff
@@ -0,0 +1,263 @@
+import React, { useState, useMemo } from 'react';
+import { useId } from '@react-aria/utils';
+import Fuse from 'fuse.js';
+import {
+  css,
+  spacing,
+  Select,
+  Option,
+  cx,
+  TextInput,
+} from '@mongodb-js/compass-components';
+import _ from 'lodash';
```

Comment:
We have troubles keeping lodash versions across packages in sync, if we can avoid using it here, I'd try to avoid that